Tools/MiniBrowser/wpe/main.cpp

```diff
@@ -82,6 +85,7 @@ static const GOptionEntry commandLineOptions[] =
 #if ENABLE_WPE_PLATFORM
     { "use-wpe-platform-api", 0, 0, G_OPTION_ARG_NONE, &useWPEPlatformAPI, "Use the WPE platform API", nullptr },
 #endif
+    { "size", 's', 0, G_OPTION_ARG_STRING, &windowSize, "Specify the window size to use, e.g. --size=\"800x600\"", nullptr },
```

Comment:
For this kind of parameters that need parsing, there's `G_OPTION_ARG_CALLBACK`, so that a callback is emitted with the string and you can parse it and initialize the global values

Reply:
Nice, will switch to that.